src/client/South/ConfigurePoints.jsx

```diff
@@ -39,19 +39,44 @@ const ConfigurePoints = () => {
   const dataSource = newConfig.south.dataSources[dataSourceIndex]
 
   const { points: pointsOrdered = [], protocol } = dataSource
-  const points = pointsOrdered.slice().reverse()
+  // add virtualIndex for each point for helping the filter
+  const points = pointsOrdered.slice().map((item, index) => ({ virtualIndex: index, ...item })).reverse()
+
+  // filter
+  const filteredPoints = filterText
+    ? points.filter(
+      (point) => {
+        // remove the virtualIndex from filterable attributes
+        const filterableAttributes = { ...point }
+        delete filterableAttributes.virtualIndex
+        return Object.values(filterableAttributes).findIndex((element) => element
+          .toString()
+          .toLowerCase()
+          .includes(filterText.toLowerCase())) >= 0
+      },
+    ) : points
 
   /**
-   * Get the index of point for reversed poitns list
-   * @param {number} index of point
-   * @returns {number} return reversed index
-   */
+  * Get the index of point for reversed poitns list
+  * @param {number} index of point
+  * @returns {number} return reversed index
+  */
   const reversedIndex = (index) => {
```

Comment:
This method is not used elsewhere. I think it can be merged with findIndexBasedOnVirtualIndex ?

Reply:
It can be merged, but i left a comment to be clear what's happening in those lines